ansible/kubernetes.yml

```diff
@@ -8,4 +8,11 @@
   roles:
     - etcd-helpers
 
+- hosts: k8s-cluster
+  tasks:
+    - name: Annotate nodes
+      command: "kubectl annotate node --overwrite {{ inventory_hostname }} {{ item.key }}={{ item.value }}"
```

Comment:
thought: I'm curious if the [community.kubernetes.k8s](https://docs.ansible.com/ansible/latest/collections/community/kubernetes/k8s_module.html) collection could be used, but as long as we are on Ansible 2.7, it's not even an option.

Reply:
So; the offline-package branch bumps to 2.9 so we could use it there. But for now I'd just keep this as it's simple enough.